src/less-style/structure.less

```diff
@@ -1,6 +1,10 @@
 // Styling for the vellum container and its main content blocks: question tree, question properties panel, etc.
 @import "lib/main";
 
+body.vellum-full-screen {
+    overflow-y: hidden;
```

Comment:
Yeah, this does make me a little nervous, but I think it's logical since fullscreen makes vellum fixed position and screen height anyway. I'm pretty sure the below-the-fold content isn't really content, it's just that a bunch of elements in HQ's layout have their height explicitly set taller than the screen.